mmv1/third_party/terraform/resources/resource_compute_region_instance_group_manager.go.erb

```diff
@@ -745,7 +751,7 @@ func resourceComputeRegionInstanceGroupManagerUpdate(d *schema.ResourceData, met
 	<% end -%>
 
 	if d.HasChange("stateful_disk") {
```

Comment:
This will cause expandStatefulPolicy to be run twice in beta currently - should be merged with the other stateful_disk block in this function.